frontend/src/components/organisms/Header.js

```diff
@@ -76,34 +88,43 @@ const Header = () => {
                 <Link inline href="/" title="Accueil - api.gouv.fr - DINUM">
                   <p className="fr-header__service-title">DataPass</p>
                 </Link>
-                <p className="fr-header__service-tagline">
-                  habilitations juridiques
-                </p>
+                {!user && (
+                  <p className="fr-header__service-tagline">
+                    habilitations juridiques
+                  </p>
+                )}
               </div>
             </div>
             <div className="fr-header__tools">
               <div className="fr-header__tools-links">
                 <ul className="fr-btns-group">
-                  {displayContactLink && (
+                  {user && !user.roles.length && (
```

Comment:
we use the fonction isEmpty for a similar test here https://github.com/betagouv/datapass/blob/e3f17bae768d9624f9a1a8ef453b781376c8ff57/frontend/src/Routes.js#L67

We should use the same evalutation everywhere to ensure strange side effect on edge cases. Ex: if roles is null or undefined.